bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -489,36 +503,56 @@ metadata:
               name: secretshare
           spec:
             serviceAccountName: secretshare
-            affinity:
-              nodeAffinity:
-                requiredDuringSchedulingIgnoredDuringExecution:
-                  nodeSelectorTerms:
-                  - matchExpressions:
-                    - key: beta.kubernetes.io/arch
-                      operator: In
-                      values:
-                      - amd64
-                      - ppc64le
-                      - s390x
             containers:
-            - command:
-              - /manager
-              args:
-              - --enable-leader-election
-              image: quay.io/opencloudio/ibm-secretshare-operator:1.2.0
-              imagePullPolicy: Always
-              name: ibm-common-service-operator
-              env:
-              - name: OPERATOR_NAME
-                value: "secretshare"
-              resources:
-                limits:
-                  cpu: 500m
-                  memory: 512Mi
-                requests:
-                  cpu: 200m
-                  memory: 200Mi
-            terminationGracePeriodSeconds: 10
+              - name: ansible
```

Comment:
@DanielXLee Do we need to change the secretshare and cs webhook in the CSV?

Reply:
I don't change any code for secretshare and webhook, This bundle manifests generated by template https://github.com/IBM/ibm-common-service-operator/blob/master/config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml, maybe I need to update this template.